### PR TITLE
Feature/discord login

### PR DESCRIPTION
**Pull request by timi-codes**

### CODE REVIEW

File: src/data/.test.channels.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (LeslieOA) [inline src/data/.test.channels.json:1]: @timi-codes I imagine there's been a `git add . --force`?
Would be good to keep this file ignored (and as the webhooks have now been exposed, let's get Fancy to cycle/update them and provide new ones).

File: build/static/js/main.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/*! For license information please see main.js.LICENSE.txt */
```
Comment (LeslieOA) [inline build/static/js/main.js:1]: @timi-codes Same with this file.
The `build` folder is also `.gitignore`'d, so let's try not to force push.

**Comment by LeslieOA** [comment]
@timi-codes @tal87 @Shivvam This is great stuff!
Holler if you aren't already owners/admins on this org/repository. 

I left a few comments on here and the Discord regarding files that should continue to be GIT-ignored (`.gitignore`) that are currently exposing test channel webhooks.

**Comment by timi-codes** [comment]
@LeslieOA Thanks for the review. All feedback has been fixed

**Comment by timi-codes** [comment]
I would also like to deploy the backend here. Can you help create a repo and give me access. Thanks @LeslieOA

**Review by LeslieOA — APPROVED** [review]
@timi-codes Until channel data is fetched from your backend, it would also be good to re-instate the example test channel file (see: https://github.com/RadarDAO-xyz/radar-signals-extension/blob/main/src/data/.test.channels.example.json).
Other than that: approved!

**Comment by LeslieOA** [comment]
> @LeslieOA Thanks for the review. All feedback has been fixed

No worries! Great work on this and the BE!
I've pinged the team about cycling webhook credentials and other things that would be good to keep secret/untracked that we'd also have to re-generate/cycle in #product-squad.

**Comment by LeslieOA** [comment]
> I would also like to deploy the backend here. Can you help create a repo and give me access. Thanks @LeslieOA

Check your email (for an invite the the organisation).
Apologies: I thought Ed had added you already. Also: are you on the `#product-squad` channel?

**Comment by Shivvam** [comment]
> > I would also like to deploy the backend here. Can you help create a repo and give me access. Thanks @LeslieOA
> 
> Check your email (for an invite the the organisation). Apologies: I thought Ed had added you already. Also: are you on the `#product-squad` channel?

Got the invite, not part of #product-squad channel now

**Comment by LeslieOA** [comment]
> > > I would also like to deploy the backend here. Can you help create a repo and give me access. Thanks @LeslieOA
> > 
> > 
> > Check your email (for an invite the the organisation). Apologies: I thought Ed had added you already. Also: are you on the `#product-squad` channel?
> 
> Got the invite, not part of #product-squad channel now

@Shivvam It still says your invitation is pending; you'll need to accept it methinks.
What's your username/handle on Radar (Discord)? I'll ping you a link to the channel there.

**Pull request merged**